.gitignore

```diff
@@ -5,3 +5,5 @@
 # See https://github.com/github/gitignore/tree/master/Global for a list of global ignore rules.
 /bazel-*
 /samples/helloworld/bazel-*
```

Comment:
Does * or ** work? Something like `/samples/*/bazel-*` or `/samples/**/bazel-*`

Reply:
I was trying to avoid a `**` since it really should only be at the roots of workspaces, but I think `/samples/*/bazel-*` is reasonable.

Done.